src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/FindT.txt

```diff
@@ -7,37 +6,37 @@ Errors: Error 0-24: The function 'Find' has some invalid arguments.|Error 14-18:
 Errors: Error 0-25: The function 'Find' has some invalid arguments.|Error 5-9: Invalid argument type (Table). Expecting a Text value instead.|Error 11-15: Invalid argument type (Table). Expecting a Text value instead.
 
 >> Find(If(false, ["blank table"], Blank()), If(false, ["blank table"], Blank()), [1])
-Blank()
+Table({Value:1})
```

Comment:
This is not right... The first (and second) argument is a blank table. For tabular functions, if a blank table is input, then the result should be blank

Reply:
Resolved by recent Carlos changes.